### PR TITLE
Support proxy-arp for service IPs

### PR DESCRIPTION
**Pull request by jojimt**

### CODE REVIEW

File: vlanBridge.go [hunk 1]
```diff
@@ -605,44 +643,24 @@ func (vl *VlanBridge) processArp(pkt protocol.Ethernet, inPort uint32) {
...
-					ethPkt.Data = arpPkt
-					log.Debugf("Sending Proxy ARP response Ethernet: %+v", ethPkt)
-
-					// Construct Packet out
-					pktOut := openflow13.NewPacketOut()
-					pktOut.Data = ethPkt
-					pktOut.AddAction(openflow13.NewActionOutput(inPort))
-
 					// Send the packet out
+					pktOut := getProxyARPResp(&arpIn, dstEp.MacAddrStr,
```
Comment (shaleman) [inline vlanBridge.go]: Can we use this function in vxlanBridge.go ARP processing too?

**Comment by shaleman** [comment]
Shouldn't we be making this change in vxlanBridge and vlrouter modes too?

**Comment by shaleman** [comment]
Changes look good. please see if we should make the same changes in other two modes as well

File: vlanBridge.go (hunk 1 above)
Comment (jojimt) [inline vlanBridge.go]: Yes, we can.

**Comment by jojimt** [comment]
We have not yet enabled service proxy in those modes. When we do that, we will add this as well.

**Pull request merged**